platform/qt/src/qmapboxgl.cpp

```diff
@@ -854,32 +998,51 @@ void QMapboxGL::addAnnotationIcon(const QString &name, const QImage &icon)
     d_ptr->mapObj->addAnnotationIcon(name.toStdString(), toSpriteImage(icon));
 }
 
-QPointF QMapboxGL::pixelForCoordinate(const Coordinate &coordinate_) const
+/*!
+    Returns the offset in pixels for \a coordinate_. The origin pixel coordinate is
+    located at the top left corner of the map view.
+
+    It will work for any coordinate, even if the coordinate is not currently visible
```

Comment:
s/It will work/This method returns the correct value/

But also note that the return value depends on the zoom level, rotation, and pitch.